cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/sla/HostFaultInjectionExample.java

```diff
@@ -53,7 +53,7 @@
  * @author raysaoliveira
```

Comment:
The name of this file must be as before. The class of the previous comment that must be renamed.